src/Uno.UI/UI/Xaml/FrameworkElement.Layout.netstd.cs

```diff
@@ -161,16 +162,19 @@ private void InnerArrangeCore(Rect finalRect)
 
 			_logDebug?.Debug($"{DepthIndentation}{FormatDebugName()}: InnerArrangeCore({finalRect}) - allowClip={allowClipToSlot}, arrangeSize={arrangeSize}, _unclippedDesiredSize={_unclippedDesiredSize}, forcedClipping={needsClipToSlot}");
 
+			var name = FormatDebugName();
+			Console.WriteLine(name);
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6128211)